gsoc-2020/extension-activity-monitor.md

```diff
@@ -0,0 +1,49 @@
+# Extension Activity Monitor 
+
+**Mentor:** Luca Greco, ...
```

Comment:
@wagnerand would you mind to review this project draft?

also, would you mind if I add you as a co-mentor on this project?

Reply:
Neat idea, sounds great! I'm more than happy to co-mentor!